.github/workflows/release.yaml

```
@@ -32,6 +32,10 @@ on:
         description: "Upload to itch.io (if configured)"
         default: true
         type: boolean
+      deploy-to-pages:
+        description: "Deploy to Github Pages (requires web build)"
```

```suggestion
      upload_to_github_pages:
        description: "Upload web build to GitHub Pages"
```

The `deploy` verb is good too, but I'd like to keep it consistent. We can either rename this, or rename `upload_to_itch` -> `deploy_to_itch`.
Should use underscores instead of hyphens either way. Also will need to make this change in other places to match.